shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -118,40 +149,15 @@ - (void)deregisterNotifications;
 @end
 
 @implementation FlutterViewController {
+  // TODO(cbracken): https://github.com/flutter/flutter/issues/137801
+  // Eliminate once we can use weak pointers in platform_view_ios.h.
   std::unique_ptr<fml::WeakNSObjectFactory<FlutterViewController>> _weakFactory;
-  fml::scoped_nsobject<FlutterEngine> _engine;
-
-  // We keep a separate reference to this and create it ahead of time because we want to be able to
-  // set up a shell along with its platform view before the view has to appear.
-  fml::scoped_nsobject<FlutterView> _flutterView;
-  fml::scoped_nsobject<UIView> _splashScreenView;
-  fml::ScopedBlock<void (^)(void)> _flutterViewRenderedCallback;
-  UIInterfaceOrientationMask _orientationPreferences;
-  UIStatusBarStyle _statusBarStyle;
+
   flutter::ViewportMetrics _viewportMetrics;
-  BOOL _initialized;
-  BOOL _viewOpaque;
-  BOOL _engineNeedsLaunch;
-  fml::scoped_nsobject<NSMutableSet<NSNumber*>> _ongoingTouches;
-  // This scroll view is a workaround to accommodate iOS 13 and higher.  There isn't a way to get
-  // touches on the status bar to trigger scrolling to the top of a scroll view.  We place a
-  // UIScrollView with height zero and a content offset so we can get those events. See also:
-  // https://github.com/flutter/flutter/issues/35050
-  fml::scoped_nsobject<UIScrollView> _scrollView;
-  fml::scoped_nsobject<UIView> _keyboardAnimationView;
-  fml::scoped_nsobject<SpringAnimation> _keyboardSpringAnimation;
   MouseState _mouseState;
-  // Timestamp after which a scroll inertia cancel event should be inferred.
-  NSTimeInterval _scrollInertiaEventStartline;
-  // When an iOS app is running in emulation on an Apple Silicon Mac, trackpad input goes through
-  // a translation layer, and events are not received with precise deltas. Due to this, we can't
-  // rely on checking for a stationary trackpad event. Fortunately, AppKit will send an event of
-  // type UIEventTypeScroll following a scroll when inertia should stop. This field is needed to
-  // estimate if such an event represents the natural end of scrolling inertia or a user-initiated
-  // cancellation.
-  NSTimeInterval _scrollInertiaEventAppKitDeadline;
 }
 
+@synthesize viewOpaque = _viewOpaque;
```

Comment:
curious why we need to manually synthesize these?

Reply:
I haven't looked at the other but presumably because we already had non-property getters and setters in the public header.

We can (and I will) follow up with a patch that migrates those to property syntax with a custom `isViewOpaque` getter, but that's orthogonal to ARC.